db/seeds.rb

```diff
@@ -19,60 +19,131 @@
-            name: "Mexican Scuba Diving", image_url: "cancun.jpg",
-            description: "Experience some of the bluest waters.")
-Trip.create(destination_id: 4, activity_id: 5, total_cost: 65000, trip_cost: 50000,
-            name: "Tianmen Mountains", image_url: "tianmen.jpg",
-            description: "The only thing more exhilarating than the climb to the top is the fall to the bottom in this remote Chinese valley.")
+image1 = File.open('app/assets/images/canadian-rockies.jpg')
+trip1 = Trip.new(destination_id: 1,activity_id: 1,name: "Canadian Rockies",
+                 description: "Navigate the trails in Alberta and British Columbia.",
+                 trip_cost: 10000.00)
+trip1.image = image1
+trip1.save
+
+image2 = File.open('app/assets/images/barrier.jpg')
+trip2 = Trip.new(destination_id: 6,activity_id: 3,name: "Great Barrier Reef",
+                 description: "Dive deep and experience on of the most colorful places on the planet.",
+                 trip_cost: 21000.00)
+trip2.image = image2
+trip2.save
+
+image3 = File.open('app/assets/images/safari.jpg')
+trip3 = Trip.new(destination_id: 5,activity_id: 6,name: "African Safari",
+                 description: "Fulfill your dreams and capture the biggest game.",
+                 trip_cost: 15800.00)
+trip3.image = image3
+trip3.save
+
+image4 = File.open('app/assets/images/base.jpg')
+trip4 = Trip.new(destination_id: 3,activity_id: 5,name: "Swiss Base Jumpers",
+                 description: "Take a gamble and dive off the cliffs of Lauterbrunnen.",
+                 trip_cost: 22550.00)
+trip4.image = image4
+trip4.save
+
+image5 = File.open('app/assets/images/climbo.jpg')
+trip5 = Trip.new(destination_id: 4,activity_id: 4,name: "Climbing in Pakistan",
+                 description: "Summit some of the most beautiful and untouched mountains on Earth.",
+                 trip_cost: 7850.00)
+trip5.image = image5
+trip5.save
+
+image6 = File.open('app/assets/images/argentina.jpg')
+trip6 = Trip.new(destination_id: 2,activity_id: 1,name: "Patagonia",
+                 description: "Travel to the end of the Earth and experience the strange beauty of Patagonia.",
+                 trip_cost: 36250.00)
+trip6.image = image6
+trip6.save
+
+image7 = File.open('app/assets/images/river.jpg')
+trip7 = Trip.new(destination_id: 1,activity_id: 2,name: "The Colorado Rapids",
+                 description: "Check an item off your bucket list with this once in a lifetime trip down the Colorado River.",
+                 trip_cost: 9550.00)
+trip7.image = image7
+trip7.save
+
+image8 = File.open('app/assets/images/basez.jpg')
+trip8 = Trip.new(destination_id: 6,activity_id: 5,name: "Base Jumping the Fjords",
+                 description: "Performing the most daring of stunts in the most isolated of places.",
+                 trip_cost: 38900.00)
+trip8.image = image8
+trip8.save
+
+image9 = File.open('app/assets/images/dolomites.jpg')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.